src/components/NcRelatedResourcesPanel/NcResource.vue

```diff
@@ -22,33 +22,44 @@
 
 <template>
 	<li class="resource">
-		<div v-tooltip="tooltip" class="resource__desc">
-			<h5>{{ title }}</h5>
-			<p v-if="subtitle">
-				{{ subtitle }}
-			</p>
-		</div>
-		<NcButton :aria-label="labelTranslated"
+		<NcButton class="resource__button"
+			:aria-label="labelTranslated"
 			type="tertiary"
 			:href="url">
 			<template #icon>
-				<ArrowRight :size="20" />
+				<div class="resource__icon">
+					<img :class="{ 'resource__icon--invert': isDarkTheme }"
+						:src="icon">
+				</div>
 			</template>
+			{{ title }}
 		</NcButton>
 	</li>
 </template>
 
 <script>
+import { imagePath } from '@nextcloud/router'
+
 import NcButton from '../NcButton/index.js'
 import Tooltip from '../../directives/Tooltip/index.js'
+import { t } from '../../l10n.js'
 
-import ArrowRight from 'vue-material-design-icons/ArrowRight.vue'
+/**
+ * Subtitles are implemented by providers with `\OCA\RelatedResources\IRelatedResourceProvider::setSubtitle()`
+ *
+ * FIXME icons should be provided by the backend https://github.com/nextcloud/related_resources/issues/15
+ */
+const subtitleIconMap = {
+	Calendar: imagePath('calendar', 'calendar.svg'),
+	'Deck board': imagePath('deck', 'deck.svg'),
+	Files: imagePath('files', 'app.svg'),
+	'Talk Room': imagePath('spreed', 'app.svg'),
```

Comment:
Totally agree with the fixme.
Also the backends should be in the respective apps, but other dicussion.

Anyway this here should be 
```suggestion
	'Talk conversation': imagePath('spreed', 'app.svg'),
```

We never use the word Room in the UI in talk

Reply:
Also this breaks when translating is fixed:
https://github.com/nextcloud/related_resources/issues/16

Reply:
Done in https://github.com/nextcloud/related_resources/pull/21 and https://github.com/nextcloud/related_resources/pull/22